experiments/compositions/samples/AttachedAKS/aks-alice.md

```diff
@@ -1,21 +1,11 @@
-# Copyright 2024 Google LLC
```

Comment:
We need a better way to fix this. Meanwhile run "make fmt" at the root directory to autogenerate these.